data-prepper-plugins/otel-trace-group-prepper/src/main/java/com/amazon/dataprepper/plugins/prepper/oteltracegroup/TraceGroupWrapper.java

```diff
@@ -0,0 +1,63 @@
+package com.amazon.dataprepper.plugins.prepper.oteltracegroup;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class TraceGroupWrapper {
+    public static final String TRACE_GROUP_NAME_FIELD = "traceGroup.name";
+    public static final String TRACE_GROUP_END_TIME_FIELD = "traceGroup.endTime";
+    public static final String TRACE_GROUP_STATUS_CODE_FIELD = "traceGroup.statusCode";
+    public static final String TRACE_GROUP_DURATION_IN_NANOS_FIELD = "traceGroup.durationInNanos";
+
+    @JsonProperty(TRACE_GROUP_NAME_FIELD)
+    private final String name;
+
+    @JsonProperty(TRACE_GROUP_END_TIME_FIELD)
+    private final String endTime;
+
+    @JsonProperty(TRACE_GROUP_DURATION_IN_NANOS_FIELD)
+    private final Long durationInNanos;
+
+    @JsonProperty(TRACE_GROUP_STATUS_CODE_FIELD)
+    private final Integer statusCode;
+
+    public String getName() {
+        return name;
+    }
+
+    public String getEndTime() {
+        return endTime;
+    }
+
+    public Long getDurationInNanos() {
+        return durationInNanos;
+    }
+
+    public Integer getStatusCode() {
+        return statusCode;
+    }
```

Comment:
Suggest always using primitives over objects when possible (long, int)

Reply:
The reason I did not use primitives is to reserve for null as otherwise in case the trace group has not been populated, the default value of int and long will be confused with the meaningful value range.

Reply:
I see your point, that's an annoying detail. I could see durationInNanos ever being `0` as unlikely, but statusCode is often `0` 😐